docs/moduleCom1DFA.rst

```diff
@@ -43,6 +43,9 @@ In the directory ``Inputs``, the following files are required:
 * release area scenario as shapefile (in Inputs/REL); multiple features are possible
   -> the release area name should not contain an underscore, if so '_AF' is added
 
+.. Note:: Release thickness can be specified in the shape file by adding an attribute called `relTh` or via the
```

Comment:
Please add:
For backwards compatibility, the attribute 'd0' also works, but we suggest to use relTh in new projects.

Reply:
Added